hypershift-operator/controllers/machineconfigserver/manifests.go

```diff
@@ -47,6 +48,23 @@ func MachineConfigServerService(machineConfigServerNamespace, machineConfigServe
 	}
 }
 
+func MachineConfigServerServicePorts() []corev1.ServicePort {
```

Comment:
Are these basically default values? I would just inline them into the reconcile functions that use them — this package is meant to contain names/locators for resources, not default values

Reply:
They are but the same value is utilized in separate code locations. Before they were just duplicated and hard coded in place. I wanted to keep from duplicating strings and instead pulling them from a constant source 

I can move them into the main package and make them more constant like